LayoutTests/fast/forms/select/mac-wk2/inactive-appearance.html

```diff
@@ -11,14 +10,15 @@
 </body>
 <script>
 
-jsTestIsAsync = true;
+if (window.testRunner)
+    testRunner.waitUntilDone();
 
 addEventListener("load", async () => {
     if (window.internals)
         window.internals.setPageIsFocusedAndActive(false);
 
     await UIHelper.renderingUpdate();
-    finishJSTest();
+    testRunner.notifyDone();
```

Comment:
```suggestion
    if (window.testRunner)
        testRunner.notifyDone();
```

(just to avoid an exception when running the test manually without WKTR)